docs/guides/monitoring/telegraf.md

```diff
@@ -0,0 +1,150 @@
+# Monitoring your CodiMD instance with telegraf
+
+CodiMD provides an easy way to monitor very basic metrics. Those can easily be
+recorded by [telegraf](https://github.com/influxdata/telegraf) for further use
+with - for example - [influxdb](https://www.influxdata.com/time-series-platform/influxdb/)
+and [grafana](https://grafana.com/).
+
+To collect the metrics, use the [http input plugin](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/http).
+It will convert all numeric values from json and store them in your database.
+
+![Example screnshot of CodiMD via Telegraf in Grafana](../images/monitoring/telegraf-grafana.png)
+
+```toml
+# /etc/telegraf/telegraf.d/codimd.conf
+[[inputs.http]]
+  interval = "1m" # strictly speaking this line is not even neccessary
+  urls = [
+    "https://your-codi-server/status" # change this!
```

Comment:
The comment should mention that it has to be removed (otherwise it's called out as syntax error)